src/source-map.ts

```diff
@@ -0,0 +1,19 @@
+import * as esprima from 'esprima'
+import {SourceMapGenerator} from 'source-map'
+
+
+export function createIdenticalSourceMap(file: string, source: string) {
```

Comment:
This function creates a source map that always maps to the same line and column.
This ensures that the 1st and the 2nd instrumented codes have the same number of lines.